pkg/scheduler/scheduler_test.go

```diff
@@ -144,7 +144,7 @@ func TestScheduler(t *testing.T) {
 	errB := errors.New("binder")
 	testNode := v1.Node{ObjectMeta: metav1.ObjectMeta{Name: "machine1", UID: types.UID("machine1")}}
 
-	table := []struct {
+	table := map[string]struct {
```

Comment:
Why change this to a map when the name string is a struct member everywhere else?